### DISCUSSION TITLE
Disable warning?

### DISCUSSION THREAD

**Discussion by mgussekloo** [marlonrichert/zsh-autocomplete#513] [Q&A] [answered]
Is there a way to disable the "no matching `directory', `file', `ancestor directory', or `recent directory' completions" line?

**Comment by marlonrichert** [comment, marked as answer]
Yes, you can add to your `.zshrc` (_after_ sourcing Autocomplete) the following to hide the message or change it to something else:
```zsh
zstyle ':completion:*:warnings' format <string>
```
